### PR TITLE
Add Logging to Telemetry API

### PR DESCRIPTION
**Pull request by jlarriba**

### CODE REVIEW

**Comment by jlarriba** [comment]
/retest

There is a timeout while trying to contact etcdserver in kuttl jobs

**Comment by jlarriba** [comment]
retest

It seems to be a temporary problem with openstackcontrolplane creating the signing CA.

**Comment by jlarriba** [comment]
recheck

File: api/bases/telemetry.openstack.org_telemetries.yaml [hunk 1]
```diff
@@ -425,6 +425,62 @@ spec:
...
+                description: Logging - Parameters related to the logging
+                properties:
+                  cloNamespace:
+                    default: openshift-logging
+                    description: CLONamespace points to the namespace where the cluster-logging-operator
+                      is deployed
+                    type: string
+                  enabled:
+                    default: true
+                    description: Enabled - Whether OpenStack Ceilometer service should
```
Comment (csibbitt) [inline api/bases/telemetry.openstack.org_telemetries.yaml:438]:
```suggestion
                    description: Enabled - Whether OpenStack logging service should
```

File: config/crd/bases/telemetry.openstack.org_telemetries.yaml [hunk 2]
```diff
@@ -425,6 +425,62 @@ spec:
...
+                description: Logging - Parameters related to the logging
+                properties:
+                  cloNamespace:
+                    default: openshift-logging
+                    description: CLONamespace points to the namespace where the cluster-logging-operator
+                      is deployed
+                    type: string
+                  enabled:
+                    default: true
+                    description: Enabled - Whether OpenStack Ceilometer service should
```
Comment (csibbitt) [inline config/crd/bases/telemetry.openstack.org_telemetries.yaml:438]:
```suggestion
                    description: Enabled - Whether OpenStack logging service should
```

**Review by csibbitt — COMMENTED** [review]
LGTM, but I'm mostly reading to see what I need to learn about golang operators. :)

File: api/v1beta1/telemetry_types.go [hunk 3]
```diff
@@ -100,6 +104,20 @@ type MetricStorageSection struct {
 	MetricStorageSpec `json:",inline"`
 }
 
+// LoggingSection defines the desired state of the logging service
+type LoggingSection struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=true
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
+	// Enabled - Whether OpenStack Ceilometer service should be deployed and managed
```
Comment (vyzigold) [inline api/v1beta1/telemetry_types.go:112]:
```suggestion
	// Enabled - Whether OpenStack logging service should be deployed and managed
```

**Comment by jlarriba** [comment]
/retest

recheck

**Comment by jlarriba** [comment]
recheck

**Pull request merged**